apps/website/src/content/docs/typography.mdx

```diff
@@ -1,17 +1,60 @@
 ---
 title: Typography
-description:
+description: Typography components provide consistency in text styling across the application.
 thumbnail: Typography
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='text--basic' />
-
 ## Text
 
-<LiveExample src='Text.main.jsx' truncate={false}>
-  <AllExamples.TextMainExample client:load />
+`Text` is a [polymorphic](#polymorphic-only-semantics-and-not-size) typography component that can be used to render any kind of text as any kind of element.
+
+```jsx
+<Text>This is a text.</Text>
+```
+
+The `Text` component includes a `variant` prop that allows you to adjust the text size. Available options range from `"headline"` to `"smaller"`, with `"body"` set as the default size. The `as` prop allows you to choose which HTML element the `Text` component should render, with `<span>` being the default.
```

Comment:
It renders a `<div>`, not a `<span>`. Actually the whole thing can be rephrased more cleanly. I've also included a link about [heading structure](https://www.w3.org/WAI/tutorials/page-structure/headings/).

```suggestion
By default, the `Text` component renders a `<div>` element with the regular `"body"` size. The visuals of this component are completely separate from its semantics, which helps ensure proper [heading structure](https://www.w3.org/WAI/tutorials/page-structure/headings/).

- **Visuals**: The `variant` prop can be used to customize the the size (e.g. `variant="subheading"`).
- **Semantics**: The polymorphic `as` prop can be used to render a different element (e.g. `as="h2"`).
```